libs/components/src/lib/components/card/card.component.ts

```diff
@@ -1,25 +1,29 @@
-import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core';
-import { prizmGetShadow, PrizmShadowDirective, PrizmShadowType } from '../../directives/shadow';
+import { ChangeDetectionStrategy, Component, HostBinding, Inject, Input } from '@angular/core';
+import { prizmGetShadow, PrizmShadowType } from '../../directives/shadow';
 import { PrizmShadowValue } from '../../directives/shadow/models';
 import { PrizmAbstractTestId } from '../../abstract/interactive';
 import { CommonModule } from '@angular/common';
+import { PRIZM_CARD_OPTIONS, PrizmCardOptions } from './card-options';
 
 @Component({
   selector: 'prizm-card',
   templateUrl: './card.component.html',
   styleUrls: ['./card.component.less'],
   changeDetection: ChangeDetectionStrategy.OnPush,
-  providers: [],
   standalone: true,
-  imports: [CommonModule, PrizmShadowDirective],
+  imports: [CommonModule],
 })
 export class PrizmCardComponent extends PrizmAbstractTestId {
-  @Input() shadow: PrizmShadowType = 'mini-bottom';
+  @Input() shadow: PrizmShadowType = this.options.shadow;
 
   @HostBinding('style.box-shadow')
   private get boxShadow(): PrizmShadowValue {
     return prizmGetShadow(this.shadow);
   }
 
+  constructor(@Inject(PRIZM_CARD_OPTIONS) private readonly options: PrizmCardOptions) {
```

Comment:
у токента тип `Partial<PrizmCardOptions>`, т.е. по итогу свойства `shadow` для L17 может и не быть
(fallback отсутствующих пропертей default?)
```suggestion
  constructor(@Inject(PRIZM_CARD_OPTIONS) options: Partial<PrizmCardOptions>) {
    super();
    this.options = { ...prizmCardDefaultOptions, ...options }
   }
```

Reply:
P.S. подсветило бы ошибку сразу при использовании  `inject`
```
private readonly options = inject(PRIZM_CARD_OPTIONS); // automatically inferred as `Partial<PrizmCardOptions>` from token
```

Reply:
Спасибо!